templates/VALIDATION.md

```diff
@@ -258,6 +260,29 @@ kind: Template
         ]
 ```
 
+### justWarning:
+When applying of the rule is not satisfied and `justWarning` is set, then validator will log that rule failed, but validation will
```

Comment:
The validation normally fails when a rule is not satisfied. The behaviour can be changed per rule by setting the `justWarning` property of the rule. The validator will then emit a warning only and the overall result of the validation will be unaffected by the rule.